liquibase-standard/SQLFileTestData.sql

```diff
@@ -0,0 +1 @@
+TESTDATA
```

Comment:
is it a right place to store test-files? @tati-qalified @MalloD12 @filipelautert

Reply:
Whops, this file should have been inside liquibase-standard/src/test/resources/liquibase/change/core/ directory (and it was afaik).  Could you fix it @tati-qalified ?